readme.md

```diff
@@ -41,6 +41,28 @@ const execa = require('execa');
 })();
 ```
 
+### Running test with custom environment variable
+
+Given the following shell command:
+
+```sh
+npm test -- --json
+```
+
+Let's take a look at how we can accomplish the same using `execa`:
+
+```js
+const execa = require('execa');
+
+(async () => {
+	const options = {env: {NODE_ENV: 'production'}};
+	const {stdout, stderr} = await execa('npm', ['test', '--', '--json'], options);
+	console.log({stdout, stderr});
```

Comment:
Indentation should be 3 spaces like the rest of the `README` for consistency.

Ideally indentation should be 2 or 4 spaces across the `README` because 3 is weird, but that should be a separate PR :)

Reply:
I did it using the online editor 🤔 ... Will fix

Reply:
Hmmm, are you sure? 
It looks tab indented like the rest to me 🙃

Reply:
Yes sorry, you're correct!